tabular/src/autogluon/tabular/models/tabular_nn/compilers/onnx.py

```diff
@@ -276,7 +276,7 @@ def transform(self, X):
 
 class TabularNeuralNetTorchOnnxCompiler:
     name = "onnx"
-    save_in_pkl = False
+    save_in_pkl = True
```

Comment:
Interesting that this works now, I remember originally NN_TORCH wasn't serializable, but maybe that was NN_MXNET and we just kept that assumption when migrating. It could have also been some of the code that has since been removed that wasn't serializable. Either way, very nice that we can save it to pickle now.